packages/sit-onyx/src/components/OnyxMobileNavButton/OnyxMobileNavButton.vue

```diff
@@ -113,6 +124,11 @@ defineSlots<{
       top: var(--top-position);
       left: 0;
       z-index: var(--onyx-z-index-navigation);
+
+      &--disabled {
```

Comment:
Would prefer to use `&:has(&__trigger:disabled) { .onyx-mobile-nav-button__headline }` and remove the custom `--disabled` class

Reply:
Edit: The flyout shoiuldn't open at all when disabled so there is actually no need to set the color here, right?